cmd/chaincode/doc/doc.go

```diff
@@ -28,6 +29,8 @@ const (
 	// this chaincode can be 'generic' if we pass in collection name to each function
 	collection = "docs"
 
+	couchDB           = "couchdb"
```

Comment:
couchDB = "couchDB"?

Reply:
“couchdb” is the literal they use for the Couch state DB. There is (currently) no other supported database that allows for indexes.